apiserver/app/apiKey.go

```diff
@@ -26,6 +28,17 @@ func getIntClaim(claims jwt.MapClaims, key string) (int, error) {
 	return 0, fmt.Errorf("Key '%v' in claims has wrong type", key)
 }
 
+func validateExp(exp int) error {
+	currentTimeEpoch := int(time.Now().Unix())
+	if exp < currentTimeEpoch {
+		return errors.New("Token has expired")
+	}
+	if exp-currentTimeEpoch > 1800 {
```

Comment:
let's make this constant (`1800`) configurable by env variable (so I can easily change it in `dev`, `test` or even `prod` environment)